.github/workflows/filename_workflow.yml

```diff
@@ -0,0 +1,45 @@
+name: aws-api-filename-test
```

Comment:
I am curious why this has to be a separate workflow file? Can this be included in the existing workflow?

Reply:
addressed below